package.json

```diff
@@ -92,7 +92,7 @@
     "@storybook/addon-links": "3.4.11",
     "@storybook/react": "4.0.0-rc.2",
     "babel-loader": "8.0.4",
-    "babel-plugin-inline-react-svg": "1.0.0",
+    "babel-plugin-inline-react-svg": "^1.1.0",
```

Comment:
Can we use an exact version for this?

Reply:
sure, this was just the result of running the command that the audit suggested